### PR TITLE
makeView()

### PR DESCRIPTION
**Pull request by Priyanka-Kasireddy**

### CODE REVIEW

File: battleship.py [hunk 1]
```diff
@@ -43,7 +44,8 @@ def makeModel(data):
 Returns: None
 '''
 def makeView(data, userCanvas, compCanvas):
-
+    userCanvas=drawGrid(data,userCanvas,data["user board"],True)
```
Comment (manasa133) [inline battleship.py]: update the separation between variables with _ or uppercase character

File: battleship.py [hunk 2]
```diff
@@ -135,13 +137,13 @@ def addShips(grid, numShips):
 Returns: None
 '''
 def drawGrid(data, canvas, grid, showShips):
-    for i in range(data["no.of rows"]):
-        for j in range(data["no.of cols"]):
+    for row in range(data["no.of rows"]):
```
Comment (manasa133) [inline battleship.py]: make the mentioned changes for variables

**Review by manasa133 — CHANGES_REQUESTED** [review]
make the mentioned changes

File: battleship.py (hunk 1 above)
Comment (Priyanka-Kasireddy) [reply battleship.py]: changed mam

File: battleship.py (hunk 2 above)
Comment (Priyanka-Kasireddy) [reply battleship.py]: changed mam

**Comment by Priyanka-Kasireddy** [comment]
done with the changes mam

**Pull request merged**